_apps/semantic-pull-requests.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Semantic Pull Requests
+description: Status check that ensures your pull requests follow the Conventional Commits spec
+slug: semantic-pull-requests
+screenshots:
+  - https://user-images.githubusercontent.com/2289/42729629-110812b6-8793-11e8-8c35-188b0952fd66.png
+  - https://user-images.githubusercontent.com/2289/42729632-11980b32-8793-11e8-9f8d-bf16c707f542.png
+authors: [zeke, bkeepers]
+repository: probot/semantic-pull-requests
+host: https://probot-semantic-pull-requests.herokuapp.com
+---
+
+> GitHub status check that ensures your pull requests follow the Conventional Commits spec
+
+Using [semantic-release](https://github.com/semantic-release/semantic-release)
+and [conventional commit messages](https://conventionalcommits.org)? Install this
+[Probot](https://probot.github.io/) app
```

Comment:
Could we nix the word probot here? It's obvious any apps on this website are probot apps, so I'd like to avoid the unnecessary repetition.

Reply:
Good catch. Fixed.